amanuensis/__init__.py

```diff
@@ -192,25 +198,35 @@ def app_config(
 #             region = app.boto.get_bucket_region(bucket_name, credential)
 #             config["S3_BUCKETS"][bucket_name]["region"] = region
 
+
 def _setup_data_endpoint_and_boto(app):
     if "AWS_CREDENTIALS" in config and len(config["AWS_CREDENTIALS"]) > 0:
-        #TODO why does it need to be the first one? (use the key value in the object instead of making it a list)
+        # TODO why does it need to be the first one? (use the key value in the object instead of making it a list)
         value = list(config["AWS_CREDENTIALS"].values())[0]
```

Comment:
@grugna the endpoints that create presigned URL's use the S3_delivery_bucket creds in the config file. So if the SES creds go first either the SES creds would need access to S3 or we have to change this part

Reply:
We probably need two clients, or to use the method auth instead of the class auth.

Reply:
sounds good. @nikoferraz then the specific aws creds need to be specified in blueprints/download_urls, resources/message, and resource/project

Reply:
@nikoferraz I saw you made the edits the poetry file but just wanted to point you here in case you missed this comment ^